canonical/ubuntu.com
# Update credentialling URLs

## Done

- Changed spelling from "credentialing" to "credentialling" to conform to British English
- Added redirects from /credentialing/* to /credentialling/*
- Added a period at the end of the header text on [/credentialling/syllabus](https://ubuntu-com-12064.demos.haus/credentialling/syllabus)
- Removed the "single exam" from [/credentialling](https://ubuntu-com-12064.demos.haus/credentialling)

## QA

- Verify that [/credentialing](https://ubuntu-com-12064.demos.haus/credentialing) redirects to [/credentialling](https://ubuntu-com-12064.demos.haus/credentialling)
- Verify that [/credentialing/syllabus](https://ubuntu-com-12064.demos.haus/credentialing/syllabus) redirects to [/credentialling/syllabus](https://ubuntu-com-12064.demos.haus/credentialling/syllabus)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
